btr-web/btr-main-app/components/bcros/CompletingParty.vue

```diff
@@ -0,0 +1,161 @@
+      return false
+    }
+
+    const hasErrors = errors.filter(errObj => errObj.path.includes(errorPath)).length > 0
+    if (hasErrors) {
+      return true
+    }
+  }
+
+  return false
+}
+
+const formSchema: CompletingPartySchemaType = CompletingPartySchema
+
+</script>
+
+<template>
+  <UForm
+    ref="completingPartyForm"
+    :schema="formSchema"
+    :state="model"
+    class="w-full"
+  >
+    <div class="mt-2 w-full border-t border-solid pt-4">
+      <p>
+        {{ $t('labels.requestOmit.completingPartyIntro') }}
+      </p>
+      <BcrosSection
+        :show-section-has-errors="hasErrors(['invididualType'])"
+        :section-title="$t('labels.requestOmit.completingParty')"
+        rounded-bot
+        :rounded-top="!editMode"
+        :border="false"
+      >
+        <div class="flex-col w-full">
+          <UFormGroup name="invididualType" eager-validation>
+            <template #default>
+              <URadioGroup
+                id="completingParty"
+                v-model="model.invididualType"
+                name="invididualType"
+                class="inline-block"
+                :options="individualTypeOptions"
+              />
+            </template>
+
+            <template #error="{ error }">
+              <p class="mt-2 text-red-500 dark:text-red-400 text-sm">
+                {{ error ? $t(error) : '' }}
+              </p>
+            </template>
+          </UFormGroup>
+        </div>
+      </BcrosSection>
+
+      <BcrosSection
+        :show-section-has-errors="hasErrors(['name'])"
+        :section-title="$t('placeholders.fullName')"
+        rounded-bot
+        :rounded-top="!editMode"
+        :border="false"
+      >
+        <div class="flex-col w-full">
+          <BcrosInputsNameField
+            id="completing-party-full-name"
+            v-model="model.name"
+            name="name"
+            :placeholder="$t('placeholders.fullName')"
+            :translate="true"
+          />
+        </div>
+      </BcrosSection>
+
+      <BcrosSection
+        :show-section-has-errors="hasErrors(['email'])"
+        :section-title="$t('registriesDetailsLabels.emailAddress')"
+        rounded-bot
+        :rounded-top="!editMode"
+        :border="false"
+      >
+        <div class="flex-col w-full">
+          <BcrosInputsEmailField
+            id="completing-party-email"
+            v-model="model.email"
+            name="email"
+            :translate="true"
+            :placeholder="$t('registriesDetailsLabels.emailAddressLC')"
+          />
+        </div>
+      </BcrosSection>
+
+      <BcrosSection
+        :show-section-has-errors="hasErrors(['certify'])"
+        :section-title="$t('labels.certifySection')"
+        rounded-bot
+        :rounded-top="!editMode"
+        :border="false"
+      >
+        <div class="flex-col w-full">
+          <UFormGroup name="certify" eager-validation>
+            <template #default="{ error }">
+              <ReviewConfirmCertify
```

Comment:
This one also has padding/margin issues that I think would be easy to solve if it was altered in the review/confirm page to use BcrosSection and then padding false here

Reply:
either that or bring out the part of the certify being used here into its own component so that you can avoid the layered margin/padding when using it here

Reply:
I've standardized on wrapping it in BcrosSection and made the changes necessary